sqlmesh/core/model/meta.py

```diff
@@ -225,8 +225,9 @@ def _properties_validator(
                 table_properties[eq_expr.this.name] = value_expr
         elif isinstance(v, dict):
             for key, value in v.items():
-                value_expr = exp.convert(value, copy=True)
-                value_expr.meta["dialect"] = dialect
+                value_expr = d.parse_one(str(value), dialect=dialect)
```

Comment:
not sure this is correct, what are the inputs you're expecting here?

Reply:
Updated